### ISSUE TITLE
is_callable incorrectly reports array as callable

### ISSUE THREAD

**Issue by calvinbaart** [peachpiecompiler/peachpie#467]
I'm having some issues in Laravel related to routing and I have narrowed it down to the following sample:

```php
$action = [
    "uses" => "",
    "controller" => ""
];

if (is_callable($action, true)) {
    echo "is_callable";
} else {
    echo "not_callable";
}
```

PeachPie output:
```
is_callable
```

PHP output:
```
not_callable
```

**Comment by jakubmisek** [comment]
Seems PHP does not behave according to the PHP documentation.

**syntax_only**
*If set to TRUE the function only verifies that var might be a function or method. It will only reject simple variables that are not strings, or an array that does not have a valid structure to be used as a callback. The valid ones are supposed to have only 2 entries, the first of which is an object or a string, and the second a string.*

**Comment by jakubmisek** [comment]
ok, PHP does not like keys in the array - keys cause the function to result in false.
